.github/dependabot.yml

```
@@ -3,7 +3,12 @@ updates:
   - package-ecosystem: 'github-actions'
     directory: '/'
     schedule:
-      interval: 'weekly'
+      interval: 'monthly'
```

```suggestion
      interval: 'weekly'
```

these are the easiest to merge - if job passes then safe to merge. you guys should just spent 5 seconds to click approve on those instead of leaving them off forever :)